src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task08.java

```diff
@@ -51,6 +53,21 @@ public static void main(String[] args) {
     static String generateTotalPriceList(int startNumberItems, double startPriceAllItems, int differentialNumberItems, double differentialSell, int sizeTotalPrice) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        String totalPriceList = "";
+        final int PROCENT = 100;
+        double pricePerItem =  startPriceAllItems / startNumberItems;
+        double sell = 0.0;
+        for(int i = 0, currentNumberItem = startNumberItems;
+            i < sizeTotalPrice;
+            i++, currentNumberItem += differentialNumberItems, sell += differentialSell) {
+            double priceWithSell = ( pricePerItem * currentNumberItem) - (pricePerItem * currentNumberItem * sell / PROCENT);
+            totalPriceList += currentNumberItem + " - "
+                    + NumberUtil.roundValueToTwoDigitsForMantissa(priceWithSell)
+                    + " with sell " + sell + "%";
+        if (i != sizeTotalPrice - 1){
+            totalPriceList += "\n";
+        }
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"